SPECS/libcap/libcap.spec

```diff
@@ -54,9 +55,10 @@ sed -i "s|pass_capsh --chroot=\$(/bin/pwd) ==||g" quicktest.sh
 %{_mandir}/man3/*
 
 %changelog
-* Sat May 09 00:21:44 PST 2020 Nick Samson <nisamson@microsoft.com> - 2.26-2
-- Added %%license line automatically
-
+*   Mon Oct 19 2020 Emre Girgin <mrgirgin@microsoft.com> 2.26-3
+-   Add `Provides: pkgconfig(libcap)` as it is not picked up by rpm automatically.
```

Comment:
The dev tooling natively supports `pkgconfig(...)`.

If the devel package for `libcap` isn't producing one despite a `.pc` file being included then it is likely symptomatic of a problem with the spec's `%build` or a problem with the toolchain build environment.

In either case I think the underlying issue should be addressed if possible rather than explicitly providing something that should be done automatically if things are correct.